front/app/containers/SearchPage/components/ExportToCsvDialog.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import gql from 'graphql-tag';
+import { graphql } from 'react-apollo';
+import { Modal } from 'react-bootstrap';
+import ThemedButton from 'components/StyledComponents/index';
+import ExportToCsvButton from './ExportToCsvButton';
+
+const SEARCH_EXPORT_QUERY = gql`
+  query SearchExportQuery($searchExportId: Int!) {
+    searchExport(searchExportId: $searchExportId) {
+      downloadUrl
+    }
+  }
+`;
+
+interface ExportToCsvDialogProps {
+  exportId?: Number;
+  data: any;
+  loading: boolean;
+  setExportId: any;
+}
+
+class ExportToCsvDialog extends React.Component<ExportToCsvDialogProps> {
+  componentWillMount() {
+    this.props.data.startPolling(250);
```

Comment:
I wasn't able to get it working, but the graphql HOC should have let me set a `pollInterval` in the options to avoid utilizing this hook. We might want to revisit that at some point.

Reply:
suggestion for how we "note" things like this for future evaluation? Create an issue like "evaluate pollInterval use" with a note about why? Also perhaps a good example of the balance of investing in solving a problem "right" vs just solving somehow and how much time to burn before just working around. Any advice / learning?